src/formatting/parse.rs

```diff
@@ -82,14 +82,19 @@ fn alphanum1(i: &str) -> IResult<&str, &str, PError> {
 }
 
 fn arg1(i: &str) -> IResult<&str, &str, PError> {
-    take_while1(|x: char| x.is_alphanumeric() || x == '_' || x == '-' || x == '.')(i)
+    take_while1(|x: char| {
+        x.is_alphanumeric() || x == ' ' || x == '%' || x == '/' || x == '_' || x == '-' || x == '.'
+    })(i)
 }
 
 // `key:val`
 fn parse_arg(i: &str) -> IResult<&str, Arg, PError> {
     map(
         separated_pair(alphanum1, cut(char(':')), cut(arg1)),
-        |(key, val)| Arg { key, val },
+        |(key, val)| Arg {
+            key,
+            val: val.trim(),
```

Comment:
Arg vals couldn't have spaces before, but they can now, so I added this trim, but if there's a nicer way to do that let me know.

Reply:
I think this is okay. Alternatively this can be done inside `arg1`.

Also, should `.fmt(key: )` be accepted? Maybe? Should this represent an empty string or a whitespace?

What do you think about requiring escaped spaces in arguments? Like `.datetime(f:%a\ %b)`.

Reply:
Maybe using single quotes would be best as it would allow spaces to be preserved:

```
.fmt(key:' ') 
.datetime(f:'%a %b')
```

Reply:
That might be the best way :+1: